cms/djangoapps/contentstore/views/dev.py

```diff
@@ -23,6 +23,11 @@ def dev_show_template(request, template):
     e.g. /template/ux/reference/container.html?name=Foo
     """
     try:
-        return render_to_response(template, request.GET.dict())
```

Comment:
I see that this view is identical to the one in `lms/djangoapps/debug/views.py`. In the spirit of the continual elimination of techdebt, could you refactor this or otherwise create a bug to do so? :)